src/lib/AST/ASTVisitor.cpp

```diff
@@ -1800,6 +1800,7 @@ class ASTVisitor
                 convertToReferenceKind(
                     D->getRefQualifier());
             I.specs0.isFinal |= D->template hasAttr<FinalAttr>();
+            I.specs1.isExplicitObjectMemberFunction |= D->isExplicitObjectMemberFunction();
```

Comment:
we may want to use `FunctionDecl::hasCXXFunctionObjectParameter` instead as `CXXMethodDecl::isExplicitObjectMemberFunction` has surprising behavior at times with respect to storage class specifiers.

Reply:
I added it here:

https://github.com/cppalliance/mrdocs/pull/541/files#diff-908f6a978cfcc9776770bcb54285299028b018608537c471f5ce7d78042a2f79R1788

Do you think I have to keep both or should I delete this line in which I use `CXXMethodDecl::isExplicitObjectMemberFunction`?

Reply:
@fpelliccioni Sorry for the belated response
> Do you think I have to keep both or should I delete this line in which I use 
only keep the line that uses `hasCXXFunctionObjectParameter`